src/components/AttachmentModal.js

```diff
@@ -212,24 +211,48 @@ function AttachmentModal(props) {
         [props.translate],
     );
     /**
-     * @param {Object} _file
+     * @param {Object} _data
+     * @returns {Boolean}
+     */
+    const isDirectoryCheck = useCallback(
+        (_data) => {
+            if (typeof _data.webkitGetAsEntry === 'function' && _data.webkitGetAsEntry().isDirectory) {
+                setIsAttachmentInvalid(true);
+                setAttachmentInvalidReasonTitle(props.translate('attachmentPicker.attachmentError'));
+                setAttachmentInvalidReason(props.translate('attachmentPicker.folderNotAllowedMessage'));
+                return false;
+            }
+            return true;
+        },
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This type of Approach is used in the entire file so used the same. we can remove this disabled Lint rule by adding `props` as a dependency rather than `props.translate`.

- Warning message from the `eslint` checker

`React Hook useCallback has a missing dependency: 'props'. Either include it or remove the dependency array. However, 'props' will change when *any* prop changes, so the preferred fix is to destructure the 'props' object outside of the useCallback call and refer to those specific props inside useCallback`

Reply:
Can we use the `useLocalize` hook instead of getting the translate prop from Onyx and update those instances, like we do [here](https://github.com/Expensify/App/blob/0f40047b279ff5ddcdab798d13d380d555789351/src/components/CountryPicker/CountrySelectorModal.js#L46)?

Reply:
@luacmartins , can I update Entrie `AttachModal.js` from `props.translate` to `translate` from `useLocalize`?

Reply:
Yea, I think that should be fine

Reply:
Done with changes